source/administration/config-settings.rst

```diff
@@ -3630,3 +3676,15 @@ Select the themes that can be chosen by users when ``"EnableThemeSelection"`` is
 +--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 | This feature's ``config.json`` setting is ``"AllowedThemes": "default"`` with options ``default``, ``organization``, ``mattermostDark`` and ``windows10`` optionally separated by commas. For example, ``["mattermostDark", "windows10"]`` |
 +--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+Display Settings  (Experimental)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Timezone
+Select the timezone used for timestamps in the user interface and email notifications.
+
+XXXXX///
```

Comment:
When it's false it hides the timezone setting in Account settings. When true it's shows the setting (so users can set their preferred timezone), but also automatically assigns a timezone the next time users connect.